.github/workflows/e2e.yaml

```
@@ -347,7 +340,7 @@ jobs:
             E2E_TARGET_NAMESPACE=default \
             e2e/readreplica
         env:
-          POD_NAME: ${{ steps.deploy_vald_readreplica.outputs.POD_NAME }}
+          POD_NAME: ${{ steps.deploy_vald.outputs.POD_NAME }}
```

actionlint (1.7.4)

property "deploy_vald" is not defined in object type {deploy_vald_readreplica: {conclusion: string; outcome: string; outputs: {pod_name: string}}; setup_e2e: {conclusion: string; outcome: string; outputs: {helm_extra_options: string; image_tags: string}}}

(expression)